docs/modules/operation/pages/thresholds/notification.adoc

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Unless we are going to give a walk through of creating a threshold notification, I don't think this page is necessary and could be removed.

Reply:
Will most users want to create a threshold notification? It might be useful to include it here so that the whole process of working with a threshold is outlined.

Reply:
Thanks. I'll update and remove that page.